source/features/reply-button.js

```diff
@@ -0,0 +1,30 @@
+import select from 'select-dom';
+import {h} from 'dom-chef';
+
+function getEventHandler(comment) {
+	return () => {
+		const body = select('.js-comment-body', comment).innerText.trim();
+		const blockquote = body.split('\n').map(line => '> ' + line).join('\n') + '\n\n';
+		const newCommentField = select('#new_comment_field');
+		newCommentField.value = blockquote;
+		newCommentField.focus();
+	};
+}
+
+export default () => {
+	select.all('.comment').forEach(comment => {
+		const replyAction = (
+			<button type="button" class="timeline-comment-action btn-link tooltipped tooltipped-nw" aria-label="Reply" onClick={getEventHandler(comment)}>
```

Comment:
`type="button"` is moot as it's the default value.

Reply:
The default value is actually `type=submit`

https://html.spec.whatwg.org/multipage/form-elements.html#the-button-element